demo/close_button.js

```diff
@@ -31,6 +31,16 @@ shakaDemo.CloseButton = class extends shaka.ui.Element {
       shakaDemoMain.unload();
     });
 
+    if ('documentPictureInPicture' in window) {
+      window.documentPictureInPicture.addEventListener('enter', (event) => {
```

Comment:
This event listener leaks when the UI is shut down or reconfigured.  Please use `this.eventManager.listen` instead:

```js
this.eventManager.listen(window.documentPictureInPicture, 'enter', () => {
```

(The use of addEventListener on this.button_ above is a bug.)

Reply:
Done!

Reply:
There is a problem with the extern...